core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1180,6 +1180,20 @@ public MetricKey build() {
           .setMetricType(MetricType.METER)
           .setIsClusterAggregated(false)
           .build();
+  public static final MetricKey CLIENT_CACHE_PAGE_READ_CACHE_TIME_NS =
+      new Builder("Client.CachePageReadCacheTimeMs")
```

Comment:
CachePageReadCacheTimeNanos

We current use `Nanos` as a keyword to detect that the counter type and format it accordingly:
https://github.com/prestodb/presto/blob/e14ba2b4e6bded261a3c71656098c8df323309bd/presto-cli/src/main/java/com/facebook/presto/cli/StatusPrinter.java#L157

Reply:
done